src/http.rs

```diff
@@ -23,6 +24,9 @@ pub fn resolve_url(url: &str, lang: &str) -> Option<String> {
     easy.get(true).unwrap();
     easy.url(url).unwrap();
     easy.follow_location(true).unwrap();
+    easy.max_redirections(10).unwrap();
+    easy.timeout(Duration::from_secs(5)).unwrap();
+    easy.max_recv_speed(10 * 1024 * 1024).unwrap();
```

Comment:
speed ? still no max download size

Reply:
> To solve this I limit both the download speed and the maximum request time,
so that the resulting data can only be about 50MB at most.

Reply:
aaaaah yes, ugly but works